src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -413,6 +413,7 @@ async def genomic_to_tx_segment(
         transcript: str | None = None,
         gene: str | None = None,
         coordinate_type: CoordinateType = CoordinateType.INTER_RESIDUE,
+        starting_assembly: Assembly = Assembly.GRCH38,
```

Comment:
Might want to update docstring to say something like "will attempt to liftover if starting assembly is GRCh37". It might be common sense, but wouldn't hurt to be explicit?